src/lektorium/install.py

```diff
@@ -0,0 +1,62 @@
+import atexit
+import contextlib
+import hashlib
+try:
+    import importlib.resources as importlib_resources
+except ModuleNotFoundError:
+    import importlib_resources
+import pathlib
+import shutil
+import subprocess
+import appdirs
+import lektorium
+
+
+def install():
+    path = importlib_resources.path(lektorium.__name__, 'client')
+    closer = contextlib.ExitStack()
+    path = closer.enter_context(path)
+    atexit.register(closer.close)
+    if (path / 'build').exists():
+        return (path / 'build')
+    return deploy(path)
+
+
+def deploy(path):
+    source_files = [s.read_bytes() for s in path.rglob('*') if s.is_file()]
+    source_hash = hashlib.md5()
```

Comment:
change `md5` for `sha256`?

Reply:
1. I don not believe in clash.
2. It is not security hash.